core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/_entryForm.jelly

```diff
@@ -27,11 +27,11 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <h1>${title}</h1>
   <div style="margin: 2em;">
-    <j:if test="${data.errorMessage!=null}">
+    <j:forEach var="error" items="${data.errors}">
```

Comment:
OK, so IIUC in https://github.com/jenkinsci/jenkins/commit/90670e05c17b34270cda9436eaf877cc72a341ec#diff-de6af001ecaa93f85c803b52f8831a99R550 `errorMessage` got enriched with an `errors` collection, but this got overlooked.

BTW, I see there we left the `errorMessage` variable in the `SignupInfo` class. Is this for backward compatibility, or is this another leftover that should also be cleaned up?